webpack.config.js

```diff
@@ -1,56 +1,53 @@
 const HtmlWebpackPlugin = require('html-webpack-plugin');
-const PORT = process.env.PORT || 3000;
+const { resolve } = require('path');
+const webpack = require('webpack');
 
 module.exports = {
-  devtool: 'cheap-module-eval-source-map',
   entry: [
     'react-hot-loader/patch',
-    `webpack-dev-server/client?http://localhost:${PORT}`,
+    'webpack-dev-server/client?http://localhost:8080',
     'webpack/hot/only-dev-server',
-    './src/index.js',
+    './index.js',
   ],
+  output: {
+    filename: 'bundle.js',
+    path: resolve(`${__dirname}/dist`),
+    publicPath: '/',
+  },
```

Comment:
I see that `` output `` is moved from bottom to top, how do you decide the order of entries in this `` module.exports `` object?